omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -55,6 +55,15 @@
       proxy_pass <%= node['private_chef']['data_collector']['root_url'] %>;
     }
     <% end -%>
+
+    <% if node['private_chef']['data_collector']['root_url'] && node['private_chef']['data_collector']['token'] -%>
+    location ~ /organizations/([^/]+)/data-collector {
+      set $request_org $1;
+      access_by_lua_file '<%= @script_path %>/check_token_using_endpoint.lua';
+      proxy_pass $upstream;
```

Comment:
When I was playing with this, I had an issue with `proxy_pass http://something/data-collector` and the regexp-match in `location ~ /organizations/([^/]+)/data-collector`. I had worked around this with something that would _here_ look like this:

```
rewrite ^/organization/(?:.*)/data-collector$ <%= URI.parse(node['private_chef']['data_collector']['root_url']).path %> break;
proxy_pass <%= URI.parse(node['private_chef']['data_collector']['root_url']).scheme %>://data-collector
```

I've just gotten the impression that the `$upstream` trick might have addressed the same issue. This would be a more localised way around it, maybe? (The less _huh?_ the better...)